<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NPlayer</class>
 <widget class="QMainWindow" name="NPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NPlayer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="margin">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="btnRun">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>26</height>
         </size>
        </property>
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnStop">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>26</height>
         </size>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnRecord">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>26</height>
         </size>
        </property>
        <property name="text">
         <string>Record</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="5">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0" colspan="7">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="Tools::StrategyView" name="graphicsView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>10</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="1" column="0" rowspan="7">
         <widget class="QPlainTextEdit" name="textLogger">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QToolButton" name="btnClearLog">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QCheckBox" name="checkShowPositions">
          <property name="text">
           <string>Positions</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="QCheckBox" name="checkShowIntermediates">
          <property name="text">
           <string>Intermediates</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1" colspan="2">
         <widget class="QCheckBox" name="chexkShowTrajectories">
          <property name="text">
           <string>Trajectories</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1" colspan="2">
         <widget class="QCheckBox" name="checkShowCaptors">
          <property name="text">
           <string>Captors</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QToolButton" name="btnSaveLogAs">
          <property name="text">
           <string>Save As</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QCheckBox" name="checkShowStrategyLog">
          <property name="text">
           <string>Strategy</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QToolButton" name="btnBlock">
          <property name="text">
           <string>Blocking Simulation</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QWidget" name="widgetReplayControl" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="margin">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Replay Speed</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderReplaySpeed">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>7</number>
         </property>
         <property name="value">
          <number>4</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="labelMatchTimer">
      <property name="text">
       <string>00:00</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QWidget" name="widgetSimuControl" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>View</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="btnSimulation">
            <property name="minimumSize">
             <size>
              <width>68</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>68</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Simulation</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnStrategy">
            <property name="minimumSize">
             <size>
              <width>68</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>68</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Strategy</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labelStrategy">
         <property name="text">
          <string>Strategy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboStrats"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labelColor">
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbColor">
         <property name="maximumSize">
          <size>
           <width>58</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labelSpeed">
         <property name="text">
          <string>Speed</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="spSpeed">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::PlusMinus</enum>
         </property>
         <property name="suffix">
          <string> m/s</string>
         </property>
         <property name="maximum">
          <double>2.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>998</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenReplay"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionStop"/>
    <addaction name="actionRecord"/>
   </widget>
   <widget class="QMenu" name="menuTable">
    <property name="title">
     <string>Table</string>
    </property>
    <addaction name="actionNew_configuration"/>
    <addaction name="actionNew_random_configuration"/>
    <addaction name="actionEmpty_configuration"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="actionSimulation"/>
    <addaction name="actionReplay"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRun"/>
   <addaction name="menuTable"/>
   <addaction name="menuMode"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRun"/>
   <addaction name="actionStop"/>
   <addaction name="actionRecord"/>
  </widget>
  <action name="actionRun">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="NPlayer.qrc">
     <normaloff>:/gui/play</normaloff>:/gui/play</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="NPlayer.qrc">
     <normaloff>:/gui/stop</normaloff>:/gui/stop</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="NPlayer.qrc">
     <normaloff>:/gui/record</normaloff>:/gui/record</iconset>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
  </action>
  <action name="actionNew_random_configuration">
   <property name="text">
    <string>New random configuration</string>
   </property>
  </action>
  <action name="actionNew_configuration">
   <property name="text">
    <string>New configuration...</string>
   </property>
  </action>
  <action name="actionEmpty_configuration">
   <property name="text">
    <string>Empty configuration</string>
   </property>
  </action>
  <action name="actionSimulation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Simulation</string>
   </property>
  </action>
  <action name="actionReplay">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Replay</string>
   </property>
  </action>
  <action name="actionOpenReplay">
   <property name="text">
    <string>Open Replay File...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Tools::StrategyView</class>
   <extends>QGraphicsView</extends>
   <header>StrategyView.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btnRun</tabstop>
  <tabstop>btnStop</tabstop>
  <tabstop>btnRecord</tabstop>
  <tabstop>graphicsView</tabstop>
  <tabstop>textLogger</tabstop>
  <tabstop>btnClearLog</tabstop>
  <tabstop>chexkShowTrajectories</tabstop>
  <tabstop>checkShowPositions</tabstop>
  <tabstop>checkShowIntermediates</tabstop>
 </tabstops>
 <resources>
  <include location="NPlayer.qrc"/>
 </resources>
 <connections/>
</ui>
